Branch-SDK/Branch-SDK/BNCCommerceEvent.h

```diff
@@ -11,16 +11,45 @@
 #import "BNCServerRequest.h"
 
 
+#pragma mark BNCProductCategory
+
+typedef NSString*const BNCProductCategory;
```

Comment:
is there supposed to be a space in `NSString*const`? Asking because I'm not sure.